modules/optimization/doc/content/modules/optimization/examples/forceInv_BodyLoad.md

```diff
@@ -0,0 +1,20 @@
+# Force Inversion Example: Dirichlet Boundary Condition
+
+## Background
+
+The MOOSE optimization module provides a flexible framework for solving inverse optimization problems in MOOSE.  This page is part of a set of examples for different types of inverse optimization problems.
+
+- [Theory](theory/InvOptTheory.md)
+- [Examples overview](examples/index.md)
+- [Example 1: Point Loads](forceInv_pointLoads.md)
+- [Example 2: Neumann Boundary Condition](forceInv_NeumannBC.md)
+- [Example 3: Distributed Body Load](forceInv_BodyLoad.md)
```

Comment:
I like the circular links for navigation so that it looks the same on every page.   The porous_flow module does this:
porous_flow/doc/content/modules/porous_flow/multiapp_fracture_flow_PorousFlow_2D.md